docs/resources/local_command.md

```diff
@@ -53,7 +62,7 @@ resource "checkmate_local_command" "example" {
 - `consecutive_successes` (Number) Number of consecutive successes required before the check is considered successful overall. Defaults to 1.
 - `create_anyway_on_check_failure` (Boolean) If false, the resource will fail to create if the check does not pass. If true, the resource will be created anyway. Defaults to false.
 - `create_file` (Attributes) Ensure a file exists with the following contents. The path to this file will be available in the env var CHECKMATE_FILEPATH (see [below for nested schema](#nestedatt--create_file))
-- `env` (Map of String) Map of environment variables to apply to the command
+- `env` (Map of String) Map of environment variables to apply to the command. Inherits the parent environment
```

Comment:
I think this is always default that "env" extends the parent process env, rather than overwriting it completely